dependencies.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from database import SessionLocal
+
+
+async def get_db() -> AsyncSession:
+    db = SessionLocal()
+
+    try:
+        yield db
```

Comment:
The `yield` statement is correctly used to provide the database session. Ensure that this function is used in an asynchronous context, such as with FastAPI, to properly handle the asynchronous session.